sysl2/sysl/visitor_test.go

```diff
@@ -274,7 +274,7 @@ func TestSequenceDiagramVisitorVisit(t *testing.T) {
 	w := MakeSequenceDiagramWriter(true, "skinparam maxMessageSize 250")
 	m, err := readModule("./tests/sequence_diagram_project.golden.json")
 	require.NoError(t, err)
-	v := MakeSequenceDiagramVisitor(l, l, w, m, "appname")
+	v := MakeSequenceDiagramVisitor(l, l, w, m, "appname", "groupby")
```

Comment:
For existing test cases:
test with `groupby = ""`

Reply:
same below.

Reply:
Taken care